src/yoti_common/anchor.processor.js

```diff
@@ -205,7 +205,7 @@ module.exports.AnchorProcessor = class AnchorProcessor {
 
     for (let i = 0; i < anchorTypes.length; i += 1) {
       const anchorType = anchorTypes[i];
-      sourceList[anchorType].forEach(yotiAnchorObj => {
+      sourceList[anchorType].forEach((yotiAnchorObj) => {
```

Comment:
I though this commit was about DocumentDetails.... these changes appear to be for anchors.  What's going on here?

Reply:
Yes this is done from ESLint recommendation.
You can see in the commit message.